inf-clojure.el

```diff
@@ -436,7 +436,8 @@ Used by this command to determine defaults."
                                             (file-name-nondirectory file-name)))
   (comint-send-string (inf-clojure-proc)
                       (format inf-clojure-load-command file-name))
-  (inf-clojure-switch-to-repl t))
+  (when current-prefix-arg
```

Comment:
It'd be better to convert this to a properly `(interactive "P")` argument. The usage of `current-prefix-arg` is quite uncommon. Guess `file-name` will have to be made optional as well, for this to work.